test/lib/ansible_test/_internal/docker_util.py

```diff
@@ -195,10 +195,19 @@ def docker_images(args, image):
     except SubprocessError as ex:
         if 'no such image' in ex.stderr:
             stdout = ''  # podman does not handle this gracefully, exits 125
```

Comment:
It looks like hitting this line would cause the json parsing code and related checks explode. it may be a good idea to exit early:
```suggestion
            return []  # podman does not handle this gracefully, exits 125
```